packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -194,16 +218,61 @@ const Autocomplete = <
     [errorMessage, hint, isOptional, label, nameOverride]
   );
 
+  const defaultValuesProp:
+    | AutocompleteValue<
+        OptionType,
+        HasMultipleChoices,
+        undefined,
+        IsCustomValueAllowed
+      >
+    | undefined = useMemo(() => {
+    if (hasMultipleChoices) {
```

Comment:
I think these should be pass as a generic to `useMemo` to work properly:
```suggestion
  const defaultValuesProp = useMemo<
    | AutocompleteValue<
          OptionType,
          HasMultipleChoices,
          undefined,
          IsCustomValueAllowed
        >
      | undefined
    >(() => {
    if (hasMultipleChoices) {
```